src/star-id.cpp

```diff
@@ -32,7 +32,7 @@ StarIdentifiers GeometricVotingStarIdAlgorithm::Go(
     StarIdentifiers identified;
     MultiDatabase multiDatabase(database);
     const unsigned char *databaseBuffer = multiDatabase.SubDatabasePointer(PairDistanceKVectorDatabase::kMagicValue);
-    if (databaseBuffer == NULL) {
+    if (databaseBuffer == nullptr) {
```

Comment:
like with `typedef`, there are dozens of uses of `NULL` throughout the codebase, if you're going to change one, change them all.